src/server/services/procedures/roboscape/robot.js

```diff
@@ -622,6 +642,25 @@ Robot.prototype.playBlinks = function (states) {
     setTimeout(step, 0);
 };
 
+/**
+ * Send a list of numbers to a robot
+ * @param {Array<Number>} states 
+ * @param {Number} delay Time each number should be shown for 
+ */
+Robot.prototype.playNumbers = function (states, delay = 4000) {
```

Comment:
Would it be better to call it something like `showNumbers` to be more consistent with the text version?

Reply:
This is replacing the existing `playBlinks` method, I didn't expose a `showNumbers` method as an RPC since only simulated robots can do it for now (we can always add it later, but we'll have to figure out what to do with simulation-only features first).

Reply:
Ah, gotcha. Makes sense!